convert2rhel/toolopts.py

```diff
@@ -338,5 +385,60 @@ def warn_on_unsupported_options():
         utils.ask_to_continue()
 
 
+def options_from_config_files(cfg_path=None):
+    """Parse the convert2rhel.ini configuration file.
+
+    This function will try to parse the convert2rhel.ini configuration file and
+    return a dictionary containing the values found in the file.
+
+    .. note::
+       This function will parse the configuration file following a specific
+       order, which is:
+       1) Path provided by the user in cfg_path (Highest priority).
+       2) ~/.convert2rhel.ini (The 2nd highest priority).
+       3) /etc/convert2rhel.ini (The lowest priority).
+    :param cfg_path: Path of a custom configuration file
+    :type cfg_path: str
+
+    :return: Dict with the supported options alongside their values.
+    :rtype: dict[str | None, str | None]
```

Comment:
Aesthetic suggestion for this docstring, nothing major. 

```suggestion
       3) /etc/convert2rhel.ini (The lowest priority).
       
    :param cfg_path: Path of a custom configuration file
    :type cfg_path: str
    :return: Dict with the supported options alongside their values.
    :rtype: dict[str | None, str | None]
```